lib/inspec/resources/file.rb

```diff
@@ -181,6 +181,34 @@ def more_permissive_than?(max_mode = nil)
       inv_mode & file.mode != 0
     end
 
+    def immutable?
+      raise Inspec::Exceptions::ResourceSkipped, "The `has_start_mode` matcher is not supported on your OS yet." unless inspec.os.unix?
```

Comment:
The error message has a copy-paste error - it says has_start_mode instead of immutable